pkg/util/util.go

```diff
@@ -45,6 +46,7 @@ func Hasher() func(string) (string, error) {
 		if err != nil {
 			return "", err
 		}
+		logrus.Infof("File %s Mode %s ModTime %s IsRegular %s\n", p, fi.Mode().String(), fi.ModTime().String(), fi.Mode().IsRegular())
```

Comment:
we might want to remove this or make it logrus.Debugf since info on every file every time we snapshot could make it difficult to parse through logs

Reply:
Oh yeah, I don't want to commit this, I just added this so I could see what was happening with these latest failures :innocent: